+         $(ENV_PIP) install -r $(ROOT)/desktop/core/base_requirements.txt; \UIREMENT_PPC64LE_FILE); \
+         echo '--- Finished $(REQUIREMENT_PPC64LE_FILE) into virtual-env'; \
          echo '--- Finished $(REQUIREMENT_FILE) into virtual-env'; \
          fi
        @$(ENV_PIP) install $(NAVOPTAPI_WHL)
diff --git a/desktop/core/base_requirements.txt b/desktop/core/base_requirements.txt
index 477002819d..f8ceb31f57 100644
--- a/desktop/core/base_requirements.txt
+++ b/desktop/core/base_requirements.txt
@@ -65,15 +65,13 @@ SQLAlchemy==1.3.8
 sqlparse==0.4.4
 tablib==0.13.0
 tabulate==0.8.9
-git+https://github.com/gethue/thrift.git
 thrift-sasl==0.4.3
-git+https://github.com/gethue/django-babel.git
 werkzeug==2.3.6  # Should remove it here and from devtools.mk
 django-utils-six==2.0
 six==1.16.0
--e file:///${ROOT}/desktop/core/ext-py3/requests-2.27.1
--e file:///${ROOT}/desktop/core/ext-py3/boto-2.49.0
--e file:///${ROOT}/desktop/core/ext-py3/django-axes-5.13.0
--e file:///${ROOT}/desktop/core/ext-py3/djangosaml2-0.18.0
--e file:///${ROOT}/desktop/core/ext-py3/eventlet-0.30.2
--e file:///${ROOT}/desktop/core/ext-py3/pysaml2-7.3.1
+-e file:////hue/desktop/core/ext-py3/requests-2.27.1
+-e file:////hue/desktop/core/ext-py3/boto-2.49.0
+-e file:////hue/desktop/core/ext-py3/django-axes-5.13.0
+-e file:////hue/desktop/core/ext-py3/djangosaml2-0.18.0
+-e file:////hue/desktop/core/ext-py3/eventlet-0.30.2
+-e file:////hue/desktop/core/ext-py3/pysaml2-7.3.1
diff --git a/desktop/core/requirements.txt b/desktop/core/requirements.txt
index 4a5833a2a9..ccdf5ce7e3 100644
--- a/desktop/core/requirements.txt
+++ b/desktop/core/requirements.txt
@@ -2,6 +2,5 @@ cryptography==41.0.1
 numpy==1.23.1
 pandas==1.4.2
 lxml==4.9.1
-PyYAML==5.4.1
--r ${ROOT}/desktop/core/base_requirements.txt
+PyYAML
